README.md

```diff
@@ -1 +1,182 @@
-# Portfolio-desktop-version
\ No newline at end of file
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#LICENSE)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Portfolio-desktop-version <a name="about-project"></a>
+
+
+**Portfolio-desktop-version** is a website that displays my Techinical skills and projects that i have done.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Header**
+- **Tag-Line**
+- **Social-Links**
+- **Recent-works**
+- **About**
+- **Footer**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Modern chrome Browser
+
+### Setup
+
+- Clone this repository to your desired folder with the following commend `git clone git@github.com:AnsarIbrahim/Portfolio-desktop-version.git`
+
+
+### Install
+
+- Type cd Portfolio-desktop-version
+- open index.html file in the browser
+
+### Usage
+
+- To run the project, execute the following command:
+  - Type cd Portfolio-desktop-version
+  - open index.html file in the browser
+
+### Run tests
+
+- To run tests, run the following command:
+  - N/A
+
+### Deployment
+
+- You can deploy this project using:
+  - Type cd Portfolio-desktop-version
+  - open index.html file in the browser
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
```

Comment:
[OPTIONAL] You can feel free to make this singular Author since you are the only author. This would make your work look more professional